pom.xml

```diff
@@ -26,9 +26,15 @@
 
         <!-- Test dependencies -->
         <dependency>
-            <groupId>org.junit.jupiter</groupId>
-            <artifactId>junit-jupiter-api</artifactId>
-            <version>5.5.2</version>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.12</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>com.github.stefanbirkner</groupId>
```

Comment:
As mentioned offline, let's remove this dependency.

Reply:
Removed and replaced with mock of `getEnvVariable()` using PowerMock